packages/server-wallet/src/wallet/wallet.ts

```diff
@@ -660,16 +661,28 @@ export class SingleThreadedWallet extends EventEmitter<EventEmitterType>
     response.channelUpdatedEvents().forEach(event => this.emit('channelUpdated', event.value));
   }
 
-  async assetTransferred(arg: AssetTransferredArg): Promise<void> {
+  async assetOutcomeUpdated({
```

Comment:
Something I spotted when doing this work, but is not really related to this PR: should this set of methods (which exist to implement the `ChainEventSubscriberInterface`) be private? It seems a little weird to me that they are a part of the Wallet API.

We could nest them all in a chainEventSubscriber property and pass that object into `this.chainService.registerChannel(...,...,)` instead of the entire wallet instance.

Reply:
cc @kerzhner